plugins/modules/monitoring/spectrum_model_attrs.py

```diff
@@ -0,0 +1,531 @@
+     default: yes
+     required: False
+     type: bool
+   name:
+     description:
+     - Model name.
+     type: str
+     required: True
+   type:
+     description:
+     - Model type.
+     type: str
+     required: True
+   validate_certs:
+     description:
+     - Validate SSL certificates. Only use if you can guarantee that you are talking to the correct endpoint and there is no man-in-the-middle attack happening.
+     type: bool
+     default: yes
+     required: False
+   attributes:
+     description:
+     - A list of attribute names and values to enforce.
+     - All values and parameters are case sensitive and must be provided as strings only.
+     required: True
+     type: list
+     elements: dict
+     suboptions:
+       name:
+         description:
+         - Attribute name OR hex ID.
+         - Currently defined names are..
+         - '                 App_Manufacturer (0x230683)'
+         - '                 CollectionsModelNameString (0x12adb)'
+         - '                 Condition (0x1000a)'
+         - '                 Criticality (0x1290c)'
+         - '                 DeviceType (0x23000e)'
+         - '                 isManaged (0x1295d)'
+         - '                 Model_Class (0x11ee8)'
+         - '                 Model_Handle (0x129fa)'
+         - '                 Model_Name (0x1006e)'
+         - '                 Modeltype_Handle (0x10001)'
+         - '                 Modeltype_Name (0x10000)'
+         - '                 Network_Address (0x12d7f)'
+         - '                 Notes (0x11564)'
+         - '                 ServiceDesk_Asset_ID (0x12db9)'
+         - '                 TopologyModelNameString (0x129e7)'
+         - '                 sysDescr (0x10052)'
+         - '                 sysName (0x10b5b)'
+         - '                 Vendor_Name (0x11570)'
+         - '                 Description (0x230017)'
+         - Hex IDs are the direct identifiers in Spectrum and will always work.
+         - To lookup hex IDs go to the UI..
+         - Locator -> Devices -> By Model Name -> <enter any model> -> Attributes tab.
+         type: str
+         required: True
+       value:
+         description:
+         - Attribute value. Empty strings should be C("") or C(null).
+         type: str
+         required: True
+'''
+
+EXAMPLES = r'''
+- name: Enforce maintenance mode for modelxzy01 with a note about why
+  community.general.spectrum_model_attrs:
+    url: "http://oneclick.url.com"
+    username: "{{ oneclick_username }}"
+    password: "{{ oneclick_password }}"
+    name: "modelxyz01"
+    type: "Host_Device"
+    validate_certs: true
+    attributes:
+      - name: "isManaged"
+        value: "false"
+      - name: "Notes"
+        value: "MM set on {{ ansible_date_time.iso8601 }} via CO {{ CO }} by {{ tower_user_name | default(ansible_user_id) }}"
+  delegate_to: localhost
+  register: spectrum_model_attrs_status
+'''
+
+RETURN = r'''
+spectrum_model_attrs_status:
```

Comment:
That is **not** what your module returns. It returns fields such as `msg`, `changed_attrs` etc. These need to be documented here.